sdk/core/azure-core/inc/azure/core/tracing/tracing.hpp

```diff
@@ -104,8 +109,8 @@ namespace Azure { namespace Core { namespace Tracing {
       virtual void AddAttribute(std::string const& attributeName, std::string const& value) = 0;
 
       /**
-       * @brief destroys an AttributeSet - virtual destructor to enable base class users to destroy
-       * derived classes.
+       * @brief destroys an AttributeSet - virtual destructor to enable base class users to
+       * destroy derived classes.
```

Comment:
You do use `{}` in these files for stuff like destructors with empty body. Use "`= default`" instead.